lib/matplotlib/backends/_backend_pdf_ps.py

```diff
@@ -37,7 +37,11 @@ def get_glyphs_subset(fontfile, characters):
     options = subset.Options(glyph_names=True, recommended_glyphs=True)
 
     # prevent subsetting FontForge Timestamp and other tables
-    options.drop_tables += ['FFTM', 'PfEd']
+    options.drop_tables += ['FFTM', 'PfEd', 'BDF']
+
+    # if fontfile is a ttc, specify font number
+    if fontfile.endswith(".ttc"):
+        options.font_number = 0
```

Comment:
This is extracting from the first font in the collection correct?

Reply:
I believe so; the count starts from 0.